cvat/apps/engine/media_extractors.py

```diff
@@ -341,6 +370,41 @@ def save_as_chunk(self, images, chunk_path):
         output_container.close()
         return [(input_w, input_h)]
 
+    def save_as_chunk_to_buff(self, frames, format_):
```

Comment:
I'm not sure the format needs to be customized. Chunk writer can write only one specific format (mp4 or zip). There are `save_as_chunk_to_buff` methods have the same signature, but different meaning of the `format_` argument.